Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/revert.py

```diff
@@ -48,7 +48,8 @@ def parser(cls, parser, loggers=None):
         # Only allow revert one commit one time, because git automatically generate mesaage will only contain one commit information
```

Comment:
I will remove this comment with any other changes that need to be made after review